beta/azure-container-apps/ReadMe.md

```diff
@@ -0,0 +1,279 @@
+
+## Prerequisites
+
+- A 1Password account with an active 1Password Business subscription or trial
+  > **Note**
+  >
+  > Try 1Password Business free for 14 days: <https://start.1password.com/sign-up/business>
+- An Azure account with permissions to create a Container App.
+  > **Note**
+  >
+  > If you don't have a Azure account, you can sign up for a free trial with starting credit: <https://azure.microsoft.com/en-us/free/>
+- [Azure CLI tool](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) with the ContainerApp Extension or access to the [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/quickstart) if performing the command driven deployment.
+
+## Getting started
+
+### Step 1: Generate credentials for automated user provisioning with 1Password
+
+1. [Sign in](https://start.1password.com) to your account on 1Password.com.
+2. Click [Integrations](https://start.1password.com/integrations/directory) in the sidebar.
+3. Choose your identity provider from the User Provisioning section.
+4. Choose "Custom deployment".
+5. Use the "Save in 1Password" buttons for both the `scimsession` file and `bearer token` to save them as items in your 1Password account. 
+6. Also use the download icon next to the `scimsession` file to save this file on your system.
+
+## Deploy 1Password SCIM Bridge to Azure Container App
+
+## Automatic Azure Deployment Steps using Azure CLI
+
+Using the Azure Cloud Shell can be easier (the assumption for the commands below is that you are using the Cloud Shell), but this can also been done directly on your system with the Azure CLI tool with the ContainerApp Extension
+
+Both methods need the Container App Extension added to the AZ tool of choice, using `az extension add --name containerapp --upgrade`
+
+1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com).
+2. Define variables for the deployment using the following example in the Cloud Shell, _(using the bash or PowerShell syntax for the commands)_. Update the values in a text editor before pasting it into the terminal:
+    - Using bash
+    ```bash
+    ResourceGroup="op-scim-bridge-rg"
+    Location="canadacentral"
+    ContainerAppEnvironment="op-scim-bridge-con-app-env"
+    ContainerAppName="op-scim-bridge-con-app"
+    ```
+    - Using PowerShell 
+    ```pwsh
+    $ResourceGroup="op-scim-bridge-rg"
+    $Location="canadacentral"
+    $ContainerAppEnvironment="op-scim-bridge-con-app-env"
+    $ContainerAppName="op-scim-bridge-con-app"
+    ```
+
+3. Create the resource group:
+
+   ```
+   az group create --name $ResourceGroup --location $Location
+   ```
+
+4. Create the Container App Environment:
+
+   ```
+   az containerapp env create --name $ContainerAppEnvironment --resource-group $ResourceGroup --location $Location
+   ```
+
```

Comment:
Should specify that this is for PowerShell.

Reply:
These two commands actually work for both PowerShell and bash, so no need to have separate code boxes.